### ISSUE TITLE
Uncaught ReferenceError: pageHasLoaded is not defined

### ISSUE THREAD

**Issue by blentle** [chipzoller/hugo-clarity#250]
**I have checked all the prerequisites below and I'm yet experiencing a problem**

**Describe the bug**

I got 'Uncaught ReferenceError: pageHasLoaded is not defined'  when I click the menu 'links' after I pull the latest submodule in my site, debug it , I found some variables have been moved to a assets/js/variables.js .

> Avoid long-winded descriptions
 - OS: [macos ]
 - Browser [e.g. chrome 76, safari 13]

how could I solve this problem for my earlier blog posts. 

many thanks.

**Comment by onweru** [comment]
@blentle, I couldn't reproduce this. Can you confirm if you're still experiencing it across browsers? If yes, please attach a recording of the same here. Thanks in advance.

**Comment by blentle** [comment]
While starting the  server (by command `hugo server`) after I pulled the latest submodule(hugo-clarity) for my blog. It occured like below
![WechatIMG396](https://user-images.githubusercontent.com/7837513/153714556-2fc8f865-6a9d-47f6-8052-5c2992c0bcc2.jpeg)

links options doesn't work like errors by  F12 console. 
debugged it , I found 2 variables have been moved from  ${myblog-path}/themes/hugo-clarity/assets/js/index.js  to variables.js (added in new version ). I copied them ,put it at the top of index.js ,it worked like :
![WechatIMG397](https://user-images.githubusercontent.com/7837513/153714768-2f0c1a95-0adb-4bd8-b654-eda4d1cad81f.jpeg)
